components/scream/src/share/scream_kokkos_meta.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef INCLUDE_SCREAM_KOKKOS_META
```

Comment:
Any better file name, file location, and namespace ideas?

Reply:
Would it be safe to use '#pragma once' ?

Reply:
I'm OK with pragma once, but I believe it's still nonstandard.

Just tbc, my original comment was referring to the higher-level question of where these type of app-unspecific Kokkos utils should live. Luca in particular I think has opinions on this kind of question, so I was directing his attention to the issue.